Makefile

```diff
@@ -8,6 +8,9 @@ IS_64_BIT ?= $(shell python -c 'from sys import maxsize; print(maxsize > 2**32)'
 
 all: environment install validate lint coverage
 
+bench:
+	pipenv run python -s bench/bench_chaum_pedersen.py
```

Comment:
We should really run all these in the folder, but I think it's simpler to keep adding lines like this for cleanliness.

I'd add `@echo 📊 BENCHMARKS` ahead of this.